x-pack/platform/plugins/shared/onechat/server/services/tools/tool_types/builtin/tool_type.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ToolType } from '@kbn/onechat-common';
+import type { BuiltinToolTypeDefinition } from '../definitions';
+
+export const getBuiltinToolType = (): BuiltinToolTypeDefinition => {
+  return {
+    toolType: ToolType.builtin,
+    builtin: true,
+  };
+};
```

Comment:
Just to keep everything consistent and avoid hardcoding things - we have a proper `builtin` tool type defined